cinema/serializers.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=63)
+    description = serializers.CharField(max_length=255)
```

Comment:
if you want to make a description not required field you need to specify this in serializer. If you want it to be required you need to fix model

Reply:
Oops, sorry...